src/pool/srv_iv.c

```diff
@@ -478,6 +496,7 @@ pool_iv_ent_copy(struct ds_iv_key *key, d_sg_list_t *dst, d_sg_list_t *src)
 				if (rc) {
 					if (rc == -DER_REC2BIG)
 						pik->pik_entry_size =
+							sizeof(*src_conns) + 
```

Comment:
Do we ever decrease pic_entry_size?

Reply:
The real entry size will be decreased during disconnection (pool_iv_conn_delete). But yes, pic_buf_size never got decreased for the moment.  Hmm, will consider how to improve this.